main_routers/workshop_router.py

```
@@ -0,0 +1,1095 @@
+    from steamworks.enums import EWorkshopFileType, ERemoteStoragePublishedFileVisibility, EItemUpdateStatus
+    
+    # 检查是否已存在上传标记
+    try:
+        if os.path.exists(content_folder) and os.path.isdir(content_folder):
+            marker_files = glob.glob(os.path.join(content_folder, "steam_workshop_id_*.txt"))
+            if marker_files:
+                marker_file = marker_files[0]
+                match = re.search(r'steam_workshop_id_([0-9]+)\.txt', marker_file)
+                if match:
+                    existing_item_id = int(match.group(1))
+                    logger.info(f"检测到物品已上传，找到标记文件: {marker_file}，物品ID: {existing_item_id}")
```

Ruff (0.14.8)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)